core/capabilities/ccip/ccip_integration_tests/ccipreader/ccipreader_test.go

```diff
@@ -812,7 +814,7 @@ func testSetup(
 		headTracker,
 		lpOpts,
 	)
-	assert.NoError(t, lp.Start(ctx))
+	servicetest.Run(t, lp)
```

Comment:
For `service.Service`s, this helper can be used to invoke `Start` and register `Close` automatically.